src/pages/SearchResults/components/MoreButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+
+const MoreButton = styled.button`
+  display: flex;
+  width: 328px;
```

Comment:
이 컴포넌트도 그렇고 작품이랑 작가 다 고정 너비인데
반응형으로 되게 바꿔주셔야할 것 같아요

```suggestion
  width: 100%;
```